torchvision/transforms/autoaugment.py

```diff
@@ -239,3 +239,87 @@ def forward(self, img: Tensor) -> Tensor:
 
     def __repr__(self) -> str:
         return self.__class__.__name__ + '(policy={}, fill={})'.format(self.policy, self.fill)
+
+
+class RandAugment(torch.nn.Module):
```

Comment:
I wonder if it makes sense to inherit from `AutoAugment` and override only the `_augmentation_space` method?

Reply:
Yes indeed. It's a direct copy-paste. The only reason I didn't make it static or inherit from AutoAugment is because I think we haven't nailed the API of the base class yet. I was thinking of keeping only the public parts visible and make changes once we add a couple of methods.